### ISSUE TITLE
docs: 

### ISSUE THREAD

**Issue by besich** [apache/apisix#5726]
### Improved or typo fixed.

this document contains the relevant chapters “Step 4: Run Test Cases”
i suggest that this chapter be separated into a markdown file.
The reasons are as follows:
*  follow the installtion step. make test will not run successfully, because it lacks the dependency.
*  keep the purity of installtion document. running test cases belongs to development documents, and the dependent installtion of test cases is very complex, so independent documents are required

### Solution

i suggest that this chapter be separated into a markdown file.

**Comment by besich** [comment]
problems encountered in executing make test to run unit tests.

execute
```shell
make ci-env-up
make ci-env-up project_compose_ci=ci/pod/docker-compose.common.yml
make init
make test
```
there are still environment dependency issues, mainly
*  about test-nginx.  suggest git submodule add https://github.com/iresty/test-nginx.git t/test-nginx
*  make test executes prove -I../test-nginx/lib -I./ -r -s t/. there are two main problems
    *  -I../test-nginx/lib: it’s not recommended to place dependencies outside the project.
    *  -I. : I feel that the introduction of this directory(.) is redundant, this to be confirmed? 
* make test report an error. Failed to get the version of the Nginx in PATH: Since the makefile can recognize my mac system, it’s can identify the env variable and execute it itself. at present, i’m executing TEST_NGINX_BINARY=/usr/local/bin/openresty prove -It/test-nginx/lib -I./ -r t/admin
* make ci-env-up is executed through docker composer. It requires machine memory. It is recommended to add a separate container for environment inspection. It should be executed once when starting. Check the environment and give a warning. For example, the prompt memory is less than 10g, but it is still executed. In this way, when various conditions are caused by insufficient memory settings, you can judge and adjust according to the prompt

In summary, it is recommended that the add make ci-env-init be checked similarly to the environment dependency.
which is only used for make test and the development and test APISIX built by ourselves, and will not have any impact on the production environment

the above suggestions are only for the implementation of make test and do not involve changing any production environment
